Share The Load/Model/User.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  User.swift
+//  Share The Load
+//
+//  Created by Christopher Hefferman on 12/01/2021.
+//
+
+import Foundation
+import RealmSwift
+
+final class User: Object {
+    private(set) var answeredQuestions = List<Int>()
+    
+    func appendAnsweredQuestion(id: Int) {
+        guard !answeredQuestions.contains(id) else { return }
+        answeredQuestions.append(id)
+    }
+    
+    /// Had a question on this in our Twitter regarding whether we should be 'removing' the Question object from Realm or just filtering out a list of id's when we access the data from Realm.
+    ///If we can talk this through on a call, then I can crack on
+    func removeAnsweredQuestion(questionAnswered: Question) {
```

Comment:
I think using the id rather than the question is still the way to go. I think we can find the question with the relevant id then remove it.

Doing this means the User object is in no way dependant on the Question object. Knowing less about other models is always a good thing.